templates/templated_email/order/payment/confirm_payment.email

```diff
@@ -7,8 +7,8 @@
 {% block plain %}
 {% include 'templated_email/shared/_header.email' %}
 {% blocktrans context "Payment confirmation email text" %}
-Thank you for your payment. To see your payment details please visit:
-{{ order_details_url }}
+Thank you for your payment.
+Your payment was successful charged.
```

Comment:
```suggestion
Your payment was successfully charged.
```

Reply:
@IKarbowiak We also have `confirm_order.mjml` template that needs to be updated accordingly.

Reply:
For every `.email` template there should also be the `.mjml` version.